client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html

```diff
@@ -713,29 +713,44 @@
                         </p>
                     </div>
 
-                    <div class="form-group">
-                        <button type="button"
-                                class="btn btn-default"
-                                data-ng-if="!ServiceContactCtrl.isEditing"
-                                data-translate="telephony_service_contact_edit"
-                                data-ng-click="ServiceContactCtrl.isEditing = true">
-                        </button>
-                        <button type="submit"
-                                class="btn btn-primary"
-                                data-ng-if="ServiceContactCtrl.isEditing"
-                                data-ng-disabled="!ServiceContactCtrl.hasChanges() || ServiceContactCtrl.isUpdating || contactForm.$invalid">
-                            <oui-loader class="mr-2"
-                                        data-ng-if="ServiceContactCtrl.isUpdating"
-                                        data-size="s">
-                            </oui-loader>
-                            <span data-translate="telephony_service_contact_apply"></span>
-                        </button>
-                        <button type="button"
-                                class="btn btn-default"
-                                data-ng-if="ServiceContactCtrl.isEditing"
-                                data-translate="cancel"
-                                data-ng-click="ServiceContactCtrl.cancelEdition()">
-                        </button>
+                    <div class="form-group mt-4">
+                        <div class="row">
+                            <div class="col-md-6">
+                                <hr/>
+                                <telephony-bulk-action data-billing-account="{{ ServiceContactCtrl.bulkDatas.billingAccount }}"
+                                                       data-service-name="{{ ServiceContactCtrl.bulkDatas.serviceName }}"
+                                                       data-custom-class="link d-block pl-0 mb-3"
+                                                       data-ng-disabled="ServiceContactCtrl.isBulkLoading"
+                                                       data-bulk-infos="ServiceContactCtrl.bulkDatas.infos"
+                                                       data-get-bulk-params="ServiceContactCtrl.getBulkParams"
+                                                       data-on-open="ServiceContactCtrl.applyChanges()"
```

Comment:
I don't understand : the directory changes must be applied to current service on bulk modal open ?